src/openrct2/ride/TrackData.cpp

```diff
@@ -5698,11 +5698,77 @@ static constexpr const rct_string_id RideConfigurationStringIds[] = {
     STR_EMPTY,                         // 266
 };
 
+// not 100% accurate
+const std::map<rct_string_id, std::vector<uint64_t>> TrackNameToTypes = {
```

Comment:
What is the point of this? Why do only some elements end up in this list. This looks wrong/the wrong thing being used.

Reply:
I couldn't find a way to find out what trackpieces are supported by a given ride type, asked around on the discord, and I got the answer that there is no easy way. This was my solution. It's not perfect, but at least a start.

Reply:
Presumably we don't need this yet until we come to add an objective type related to track pieces.

Reply:
That's true yeah, I forgot to remove it.